lte/gateway/docker/docker-compose.override.yaml

```diff
@@ -16,3 +16,4 @@ services:
     build:
       context: ${BUILD_CONTEXT}
       dockerfile: feg/gateway/docker/go/Dockerfile
+    image: gateway_go
```

Comment:
The PR description says this was renamed from `agw_gateway_go` to `gateway_go`. I don't understand where the `agw_gateway_go` comes from.

Reply:
It's dockers way of nameing its images: `<project>_<service>`, where the `<project>` variable is defined in `.env` by `COMPOSE_PROJECT_NAME=agw`